docs/metrics-events.md

```diff
@@ -198,13 +197,13 @@ View name|Description
 `signin`|The sign-in form
 `confirm-signin`|Displayed while awaiting sign-in confirmation
 `signin-confirmed`|The tab the user signed in from, after sign-in confirmation
-`verify-email`|The tab the user attempted to verify a signup in, before verification
```

Comment:
I think we may have over-laboured describing the different states for verification/confirmation in this table, we should prune it down to just the essentials I think. I started with this one, may do others in future edits.